src/Workspaces/Core/Portable/FindSymbols/SyntaxTree/SyntaxTreeIndex_Persistence.cs

```diff
@@ -65,11 +80,28 @@ public override void WriteTo(ObjectWriter writer)
             }
         }
 
+        var interceptsLocationInfoCount = reader.ReadInt32();
+        Dictionary<InterceptsLocationData, TextSpan>? interceptsLocationInfo = null;
+        if (interceptsLocationInfoCount > 0)
+        {
+            interceptsLocationInfo = [];
+
+            for (var i = 0; i < interceptsLocationInfoCount; i++)
+            {
+                interceptsLocationInfo.Add(
+                    new InterceptsLocationData(
+                        ImmutableCollectionsMarshal.AsImmutableArray(reader.ReadByteArray()),
```

Comment:
This is probably just me, but I'd find this easier to understand if the Read* calls were done in separate stmts on their own line, just to indicate that there is an exact mapping between the read/write behaviors.

Reply:
gotcha... i like this.

Reply:
ok.  i do like this form and would prefer to keep it.

Reply:
Think we're going to have to agree to disagree on this one. Not that big of a deal though.